src/daemon/rpmostreed-sysroot.cxx

```diff
@@ -546,17 +546,6 @@ rpmostreed_sysroot_init (RpmostreedSysroot *self)
 
   self->monitor = NULL;
 
-  /* Only use polkit when running as root on system bus; self-tests don't need it */
```

Comment:
A little bit unclear to me why we don't need this check anymore or what replaced it.

Reply:
Thanks for looking!  The more eyes on this the better.  But this logic did move to here:
https://github.com/coreos/rpm-ostree/pull/3555/files#diff-a07e75b7bf29db108edbe65cd88cb3bb7eac0694b9b165086455c103afa05da7R953
For the "am on session bus" check.  Then the remaining logic always calls this method first.